docs/source/gui.rst

```diff
@@ -6,16 +6,224 @@ Graphical User Interface
 
 The Waveform Editor provides a Graphical User Interface (GUI), with which you can:
 
-* Load an existing YAML file from disk
-* Create/remove groups
-* Create/remove waveforms
-* View or edit waveforms
-* Store changes back to YAML
+* Create, load, edit, and save waveform configurations from YAML files.
+* View plots of one or multiple waveforms simultaneously for comparison.
+* Edit waveforms through a text editor with live plot updates and error
+  highlighting.
+* Export waveform data to various formats, including IDS, CSV, PNG, and PCSSP XML.
+* Graphically edit and design the desired plasma shape using NICE.
 
-The GUI can be launched using:
+Launching the GUI
+-----------------
+
+The GUI can be launched using the following command in your terminal:
+
+.. code-block:: bash
+
+    waveform-editor gui
+
+This normally opens automatically in your default web browser. If it does not, 
+you can manually open the app in a web browser by going to the address printed in the terminal.
+For example, when the ``waveform-editor gui`` command has the following output, the app is running
+on the web address ``http://localhost:38895``.
+
+.. code-block:: console
+
+    $ waveform-editor gui
+    [...]
+    Launching server at http://localhost:38895
```

Comment:
Let's put this in a `.. note::`? That probably looks nicer in the generated docs :)